src/client-apps/nuxeo-drive/nuxeo-drive-installation-configuration.md

```diff
@@ -68,43 +68,12 @@ If you try to synchronize a folder and you haven't installed the Nuxeo Drive cli
 
 #### Installing Nuxeo Drive on GNU/Linux
 
-1. Extendented Attributes
+1. Download the binary (`.AppImage` file) from the **Nuxeo Drive** tab in the **Home** or from the [Nuxeo Drive update site](https://community.nuxeo.com/static/drive-updates/nuxeo-drive-x86_64.AppImage).
```

Comment:
In Web UI, "Home" is "User Settings", that's probably what we want to refer to.
So you'll need to update Web UI, or maybe you have?

Reply:
Yes, the patch is ready.